common/browsered/field-validation-checks.js

```diff
@@ -14,31 +14,32 @@ const validationErrors = {
   validEmail: 'Please use a valid email address',
   https: 'URL must begin with https://',
   belowMaxAmount: `Choose an amount under £${MAX_AMOUNT.toLocaleString()}`,
-  sortCode: `Sort codes must contain 6 digits`,
-  accountNumber: `Account numbers must contain 6-8 digits`
+  sortCode: 'Sort codes must contain 6 digits',
+  accountNumber: 'Account numbers must contain 6-8 digits',
+  requiresAuthorisation: 'Please authorize'
```

Comment:
I think this should be isChecked too. Also the message should be more informative

Reply:
Thanks, changed the method to `isChecked` (with passing the whole `field` object), but also if we use generic message it will not be clear in the actual requires authorisation error, that is why we also moved the `Please authorize` error to `data-error-label` in the njk

Reply:
Also, it is nice to do some refactoring of field-validation-checks.js and fix all the methods at some point. They all should return true and false and then the error should be displayed from somewhere else, currently, they are doing two things